events.md

```diff
@@ -4,7 +4,18 @@ author_profile: false
 title: Events
 ---
 
-Coming soon: this page will have a list of past and upcoming Spack talks,
-tutorials, webinars, and other events of interest.
+## 2020
 
-**For now, check out our page on [Spack at SC19](/spack-at-sc19/)**.
+### [ISC High Performance](https://www.isc-hpc.com/)
+<small>21-25 June 2020 | Frankfurt, Germany (PENDING)</small>  
+The ISC (formerly known as the International Supercomputing Conference) caters to the hardware and software demands of global research centers and businesses in the fields of HPC, artificial intelligence, machine learning and data analytics.
+
+### [ASC S3C](https://s3c.sandia.gov/)
+<small class="pull-right">14-16 April 2020 | Albuquerque, NM (POSTPONED)</small>  
+The Tri-lab Advanced Simulation & Computing Sustainable Scientific Software Conference connects individuals and teams from the ASC Tri-labs that are working to deliver scientific software solutions in a sustainable manner. Greg Becker will be presenting a session on Spack.
```

Comment:
a 20-minute presentation and a half-day tutorial

Reply:
Added in a21b820